python/main.py

```diff
@@ -18,15 +21,60 @@
     allow_headers=["*"],
 )
 
+"""
+Gets the list of all items
+"""
 @app.get("/")
 def root():
-    return {"message": "Hello, world!"}
+    # get the list of all items in the database
+    items = database.get_items()
+    # format the list and return
+    return format_items(items)
 
+"""
+Gets item with the given item_id
+"""
+@app.get("/items/{item_id}")
+async def get_item_by_id(item_id):
+
+    item = database.get_id_by_id(item_id)
+    if len(item) == 0:
+        logger.debug(f"Item not found: {item_id}")
+        return f"Item not found: {item_id}"
+    
+    return format_items(item)
+
+
+"""
+Create a new item with the given name, cateogry, image
+"""
 @app.post("/items")
-def add_item(name: str = Form(...)):
-    logger.info(f"Receive item: {name}")
+def add_item(name: str = Form(...), category: str = Form(...), image: str = Form(...)):
+
+    logger.info(f"Receive item: name = {name}, category = {category}, image = {image}")
+    
+    # remove the "@" in the beggining
+    image = image[1:]
```

Comment:
If we get request like 
```
curl -X POST \
  --url 'http://localhost:9000/items' \
  -d 'name=jacket' \
  -d 'category=fashion' \
  -d 'image=@images/default.jpg'
```
Then we have image = "@images/default.jpg" with "@" in the beginning.
But it does not work as a file path with "@", so I had to remove that character.
Am I missing something? I think there is an better way...